include/openssl/pkcs7.h

```diff
@@ -36,7 +36,7 @@ Key_Encryption_ID       rsaEncryption
 */
 
 typedef struct pkcs7_issuer_and_serial_st {
-    X509_NAME *issuer;
+    const X509_NAME *issuer;
```

Comment:
Don't change it for public structs.

Reply:
I thought there is some freedom to adapt types where useful for v3.0, or am I wrong on this?